globalVars.py

```diff
@@ -5,3 +5,4 @@
 
 gameVersion = 2.00  # Game version
 appMain = None
+stressrelease = False # Set this to true to indicade that you're playing stress release.

```

Comment:
Based on the existing implementations, the correct way is making a new class that inherits ModeHandlerBase and having the new class do all the mode-specific customizations.